src/platforms/common/configuration/sampling.mdx

```diff
@@ -41,6 +41,30 @@ Changing the error sample rate requires re-deployment. In addition, setting an S
 
 </Note>
 
+<PlatformSection supported={["python"]}>
+
+### Dynamically Sampling Error Events
+
+It is also possible to sample error events dymanically, by setting the <PlatformIdentifier name="error-sampler" /> to a function that returns the desired sample rate for the event. The <PlatformIdentifier name="error-sampler" /> receives two arguments, the <PlatformIdentifier name="event" /> and the <PlatformIdentifier name="hint" />, when called, which it can inspect in order to inform the sampling decision.
```

Comment:
```suggestion
It is also possible to sample error events dynamically, by setting the <PlatformIdentifier name="error-sampler" /> to a function that returns the desired sample rate for the event. The <PlatformIdentifier name="error-sampler" /> receives two arguments, the <PlatformIdentifier name="event" /> and the <PlatformIdentifier name="hint" />, when called, which it can inspect in order to inform the sampling decision.
```